pkg/cmd/pipelinerun/describe.go

```diff
@@ -97,6 +99,11 @@ func printPipelineRunStatus(w *tabwriter.Writer, pr *v1alpha1.PipelineRun, p cli
 	fmt.Fprintf(w, statusBody, formatted.Age(*pr.Status.StartTime, p.Time()),
 		formatted.Duration(pr.Status.StartTime, pr.Status.CompletionTime),
 		formatted.Condition(pr.Status.Conditions[0]))
+
+	if (hasFailed(pr.Status.Conditions[0]) && (pr.Status.Conditions[0].Message != "")) {
```

Comment:
Would it better to have `func hasFailed(pr) (bool, message)` , something like this?
```
func HasFailed(status v1alpha1.PipelineRunStatus) (bool, string) {
	c := status.Conditions[0]
	if c.Status == corev1.ConditionFalse {
		return true, c.Message
	}

	return false, ""
}
```
In this way, it could give a message as well and exactly functionality is also required in pipeline logs case?

Reply:
Would work I think :angel: